config/pyqcm_sse3.sh

```diff
@@ -0,0 +1,20 @@
+MODULE_RUNTIME_DEPS="
+    scipy-stack/2023b
+"
+MODULE_BUILD_DEPS="
+    cmake/3.23.1
+    cuba/4.2.2
+    eigen/3.3.7
```

Comment:
This is an avx2 module, this config should be deleted in favor of `[pyqcm.sh](https://github.com/ComputeCanada/wheels_builder/pull/104/files#diff-7a77e2bea1b285cbe10017bd62663ce5eaee5782fbe3ce0a64cad6ecac088139)`